src/components/book.js

```diff
@@ -0,0 +1,33 @@
+const Book = () => (
+  <div className="book-container">
+    <div className="book">
+      <div className="book-details">
+        <div className="book-titles">
+          <p className="category">Biography</p>
```

Comment:
- [ ] You have done well by creating the `Book` component. All that is left is to ensure that it takes the book data as a prop from which it can render the title and the author 👍

Reply:
Thank you :+1: